src/settings/settings.scss

```diff
@@ -0,0 +1,70 @@
+$backgroundColorForSection: rgba(0, 0, 0, 0.45);
```

Comment:
We can talk about that today but I would propose another way to call CSS variables.